### DISCUSSION TITLE
Sunbrid RC latest release notes

### DISCUSSION THREAD

**Discussion by vinukumar-vs** [Sunbird-RC/community#351] [General]
We are using Sunbird RC as part of Sunbird-ED release. As part of sunbird-ed release notes we want to publish the release-tags for Sunbird-RC that we used. 
As an adopter of Sunbird-RD where can we find the latest release of Sunbird-RC repo's. So that we can refer that release notes to take the release tags.

Today it's points to the  Sunbird-RC-Core Github release from the microsite (referring https://docs.sunbirdrc.dev/ => Links => Releases & Changelogs)
https://github.com/Sunbird-RC/sunbird-rc-core/releases

The above link helps to know the latest release tag of Sunbird-RC core(Registry). What about for other repositories like Certificate API, Admin etc.

**Comment by parthlawate** [comment]
@tejash-jl can you help here ?

**Comment by tejash-jl** [comment]
@vinukumar-vs all the services are released under the same tag
